charts/renga/charts/gitlab/values.yaml

```diff
@@ -3,8 +3,8 @@
 # Declare variables to be passed into your templates.
 
 image:
-  repository: gitlab/gitlab-ce
-  tag: latest
+  repository: rengahub/gitlab
+  tag: development
```

Comment:
can't we go with the standard image?

Reply:
no we can't for now

Reply:
yes you can, you just need to paste the content of `Auto-DevOps.gitlab-ci.yml` in a cm and mount it...